src/iot/serializers.py

```diff
@@ -1,6 +1,5 @@
 from datapunt_api.rest import HALSerializer
 from rest_framework import serializers
-from rest_framework.fields import DateField, JSONField
```

Comment:
Let's add autoflake to the makefile linting, it removes unused imports and variables automatically. 

in makefile lint:
`autoflake src tests --check --recursive --in-place --remove-unused-variables --remove-all-unused-imports`

in makefile lintfix:
`autoflake src tests --recursive --in-place --remove-unused-variables --remove-all-unused-imports`

Reply:
And while we're at it, I am a big fan of adding pylint as linter too.  (be ready for some more refactoring/fixing to satisfy it though). 

**makefile lint:**
`pylint --rcfile=pyproject.toml src/`

**pyproject.toml**
```
[tool.pylint.'MESSAGES CONTROL']
max-line-length = 120
disable = [
    "logging-fstring-interpolation",
    "missing-class-docstring",
    "missing-module-docstring",
    "missing-function-docstring",
    "no-else-return",
    "no-else-continue",
    "too-few-public-methods",
    "too-many-instance-attributes",
    "fixme",
    "no-name-in-module",
]
```

Reply:
Lastly, I would opt for adding mypy typechecking to the repo but after a quick test I see that it will lead to even more fixing work, so perhaps that can have it's own PBI instead. What do you think?